src/main/resources/templates/db_scripts/hridSettings.sql

```diff
@@ -1,17 +1,20 @@
-CREATE EXTENSION IF NOT EXISTS "pgcrypto";
-
--- ensure that there will be one row only
+-- Ensure that there will be one row only.
 ALTER TABLE ${myuniversity}_${mymodule}.${table.tableName}
   ADD COLUMN IF NOT EXISTS
     lock boolean DEFAULT true UNIQUE CHECK(lock=true);
+-- Derive id from tenant, module and table, setting UUID version to 4 and
+-- UUID variant byte to 8.
```

Comment:
Does this mean that the database generated UUID is a custom design specific to FOLIO?

Reply:
The pattern for microservices is that the client provides the id.
This doesn't work when a database trigger or some internal process creates new records. Then the database needs to come up with some id. One possibility is to use the md5 trick because a md5 hash and a uuid have the same number of bytes.
Another possibility is to simply use a hard-coded uuid in hridSetings.sql.

Reply:
@julianladisch 

> The pattern for microservices is that the client provides the id.

Unfortunately, that is not possible in this case because these records are required to be created without any involvement from a client.

> Another possibility is to simply use a hard-coded uuid in hridSetings.sql.

I really like this idea. I much prefer that to trying to invent our own UUID generation scheme.